script.sh

```diff
@@ -0,0 +1,17 @@
+/home/peiyuan_zhang_ntu_edu_sg/miniconda3/envs/sglang/bin/python -m sglang.launch_server \
+--model-path lmms-lab/llama3-llava-next-8b \
+--tokenizer-path lmms-lab/llama3-llava-next-8b-tokenizer \
```

Comment:
Is it possible to merge the tokenizer files `lmms-lab/llama3-llava-next-8b-tokenizer` into `lmms-lab/llama3-llava-next-8b`?
For typically text-only models, users only need to specify a `--model-path`. Now it is error-prone to specify both.

Reply:
Let me try it, but my experience is this would affect the normal pipeline inference. So there may involve more code changes in llava side.